includes/class-db.php

```diff
@@ -114,4 +115,40 @@ public static function migrate_to_1_0_0() {
 			"DELETE FROM {$wpdb->commentmeta} WHERE meta_key LIKE 'semantic_linkbacks_%'"
 		);
 	}
+
+	/**
+	 * Author Migration tool from mf2_author
+	 *
+	 * @since 5.0.0
+	 *
+	 * @return void
+	 */
+	public static function mf2_author_migration() {
+		$comments = get_comments(
+			array(
+				'fields'     => 'ids',
+				'meta_query' => array(
+					array(
+						'key'     => 'mf2_author',
+						'compare' => 'EXISTS',
+					),
+					array(
+						'key'   => 'protocol',
+						'value' => 'webmention',
+					),
+				),
+			)
+		);
+		foreach ( $comments as $comment_id ) {
+			$author = get_comment_meta( $comment_id, 'mf2_author', true );
+			if ( is_array( $author ) ) {
+				if ( array_key_exists( 'url', $author ) ) {
+					$comment               = get_comment( $comment_id, ARRAY_A );
+					$comment['author_url'] = $author['url'];
+					wp_update_comment( $comment );
+				}
+				delete_comment_meta( $comment_id, 'mf2_author' );
```

Comment:
Let's do not delete things for now ☺️

Maybe in a later version if we are sure we have anything migrated successfully!

Reply:
That's fine. Otherwise, does this look like a good idea or should I make some additional checks? Like comparing the URL to the source_url?